deploy/1_deploy_market_contracts.ts

```diff
@@ -1,26 +1,31 @@
+import { Logger, LogLevel } from '@ethersproject/logger';
+import type { DeployFunction } from 'hardhat-deploy/types';
+
+import { deployMarketContracts, saveDeployments } from '../utils/deploy';
+import { getContractsFromDeployments } from '../test/helpers/index';
+
 import {
   configureDeploymentSettings,
   validateDeployment,
   verifyContracts,
   writeContractsConfig,
   pushContractsToEthernal,
-  seedContracts,
   addContractsToDefender,
 } from '@/utils/deploy';
-import { Logger, LogLevel } from '@ethersproject/logger';
-import { DeployFunction } from 'hardhat-deploy/types';
-import { deployMarketContracts, Contracts, saveDeployments } from '../utils/deploy';
-import { getContractsFromDeployments } from '../test/helpers/index';
 
 export const deploy: DeployFunction = async (env) => {
-  const hre: CustomHardHatRuntimeEnvironment = env as unknown as CustomHardHatRuntimeEnvironment;
+  const hre: CustomHardHatRuntimeEnvironment =
+    env as unknown as CustomHardHatRuntimeEnvironment;
   Logger.setLogLevel(LogLevel.DEBUG);
   hre.log(`1_deploy_market_contracts`);
   validateDeployment({ hre });
   await configureDeploymentSettings({ hre });
   const dependentContracts = await getContractsFromDeployments(hre);
-  const contracts = await deployMarketContracts({ hre, contractNames: ['FIFOMarket', 'Certificate', 'Removal'], contracts: dependentContracts });
-  await seedContracts({ hre, contracts });
```

Comment:
getting rid of this might break our cypress tests that rely on things being seeded